content/modules/mobile-travel-tutorial/partials/installation/manual.adoc

```diff
@@ -57,47 +55,47 @@ bash git --version
 You should see the version of git installed.
 ****
 
-
 == Workshop Repo
 
-
 * Clone the "master" branch of the workshop source from GitHub. We are doing a shallow pull with `depth` as 1 to speed the cloning process.
 +
 [source,bash]
 ----
 git clone -b master --depth 1 https://github.com/couchbaselabs/mobile-travel-sample.git
 ----
 
-
 == Couchbase Server
 
+In this lesson, you will install and launch v7.1.x of Couchbase Server.
+
+Screenshots below apply to Couchbase Server 7.1.1 and there will be some variation with CBS 7.0.x. However, equivalent functionality is available in 7.0.x.
```

Comment:
State "However, equivalent functionality is available in earlier versions of server. Refer to documentation for compatible server versions"